website/src/releases/2022-09/readable-page-title.mdx

```diff
@@ -0,0 +1,15 @@
+---
+date: 2022-09-29
+title: Readable page title component
+description: We now have a component that can be used to overwrite the default document's `<title>`, to provide improved support for a Human-readable page title
+type: feature
+topics:
+  - Merchant Center
+  - UI Components
+---
+
+We now have a [`Human-readable page title`](/development/human-readable-page-title) feature describing as detailed as possible,
+the location or context of the page a user is currently on.
+
+You can now use the [`<ApplicationPageTitle>`](/api-reference/commercetools-frontend-application-shell#applicationpagetitle) component to overwrite the default page title by providing more details about a page by passing an `additionalParts` props,
+an array of strings carrying different descriptions of the features of the page. This provides more readability and context to the page title.
```

Comment:
Since we already have the documentation pages, we can simply link to them.

```suggestion
Custom Applications use a more readable document's title by default, which can be customized. You can read more about this feature in the [`Human-readable page title`](/development/human-readable-page-title) documentation.

As always, if you have questions or feedback you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```

Reply:
Thank you. 
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2793/commits/056508e8f1a58425cb3cf3a4196c7af1db312ed7